resources/iam-instance-profile-roles.go

```diff
@@ -61,6 +70,16 @@ func (e *IAMInstanceProfileRole) Remove() error {
 	return nil
 }
 
-func (e *IAMInstanceProfileRole) String() string {
```

Comment:
Please keep the `String()` function, as it is needed for identifying the resource.

Reply:
done